runner/runner.go

```diff
@@ -32,7 +34,7 @@ const (
 )
 
 func (p ProcessState) IsDone() bool {
-	return p == UNKNOWN || p == COMPLETE || p == FAILED || p == ABORTED || p == TIMEDOUT
+	return p == COMPLETE || p == FAILED || p == ABORTED || p == TIMEDOUT
```

Comment:
From the outside, the status of a given Run ID can change from UNKNOWN -> RUNNING.

Done should mean "this Run will never change state".